seatunnel-connectors-v2/connector-starrocks/src/main/java/org/apache/seatunnel/connectors/seatunnel/starrocks/util/SchemaUtils.java

```diff
@@ -138,6 +138,15 @@ public static void applySchemaChange(
         try (Statement statement = connection.createStatement()) {
             log.info("Executing change column SQL: " + changeColumnSQL);
             statement.execute(changeColumnSQL);
+        } catch (SQLException e) {
+            // from v3.3.2 onwards, StarRocks supports renaming columns
+            // if use previous starrocks version, program will be terminal
+            // suggest the sqlexception. programe will continue running
+            log.warn(
+                    "Executing change column SQL: {},  Error Message: {}",
+                    changeColumnSQL,
+                    e.getMessage());
+            e.printStackTrace();
```

Comment:
Why we need let job continue running not end with FAILED? It will let user confused.

Reply:
if not catch sqlexception.  program terminal. so previous starrocks version cann't use seatunnel's connector starrocks.
i suggest write a tip about previous starrokcs version cann't rename column. but other function is no normal.

Reply:
many compnay should use previous starrock version. but want to use seatunnel sync function

Reply:
If the rename operation fails, does it mean that some field data in the upstream cannot be synchronized to StarRocks normally? If the task does not fail at this time, how can the user perceive it?

Reply:
yes, it did. some field data in the upstream cann't be synchronized to starrocks. 
i suggest to write a note in the document that versions prior to starrocks 3.3.2 do not support renaming columns, and renamed columns will not synchronize data. error messages will also be printed in the log.   Other functions are normal

Reply:
This modification will enable the previous version of Starrocks to also use the synchronization function of SeaTunnel, without affecting the use of Starrocks versions 3.3.2 and later

Reply:
I think we can precheck the starrocks version, when the version is less than 3.3.2, we can skip execute alter table field name. Instead of catch exception when sql execute failed.

Reply:
ok, i agree it. can i contribute it?

Reply:
Sure!